src/styles.css

```diff
@@ -578,7 +577,6 @@
 
   /* Cover Page Component */
   .cio-cover-question-container--with-image {
-    justify-content: center;
```

Comment:
Since the same style comes from `cio-container--with-image`, I believe it's redundant here. Let me know if there is an edge case this handles